lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb

```diff
@@ -90,6 +90,32 @@ class SmartStandalonePatientAppGroup < Inferno::TestGroup
                ]
              }
            }
+
+      test do
+        required_suite_options(G10Options::US_CORE_7_REQUIREMENT)
+
+        id :g10_us_core_7_smart_version_check
+        title 'US Core 7 requires SMART App Launch 2.0.0 or above'
+        description %(
+          The [US Core 7 SMART on FHIR Obligations and
+          Capabilities](https://hl7.org/fhir/us/core/scopes.html) require SMART
```

Comment:
Should we use version sepcific url here https://hl7.org/fhir/us/core/STU7/scopes.html